content/en/developers/dogstatsd/_index.md

```diff
@@ -64,10 +65,54 @@ By default, DogStatsD listens on UDP port **8125**. If you need to change this,
 
 2. [Restart your Agent][10].
 
+#### DogStatsD and Docker
+
+Libraries that communicate over HTTP using the [Datadog API][11] don't require any special configuration with regard to Docker. However, applications using libraries that integrate with DogStatsD or StatsD need the library configured to connect with the Agent. Each library handles this configuration differently, so refer to the individual library's documentation for more details.
+
+After your code is configured, run your custom application container using [the `--link` option][12] to create a network connection between your application container and the Datadog Agent container.
+
+##### Example: Monitoring a basic Python application
+
+To monitor your application, run the Datadog container using the [Basic Docker Agent usage][13] instructions. The `docker run` command sets the name of the container to `dd-agent`.
+
+Next, instrument your code. Here's a basic Flask-based web application:
+
+```python
```

Comment:
We shouldn't display a full code example here since we give instructions on how to setup a DogStatsD client below. Maybe should we change this into an array of which parameter should be set for which language with which value.